testdata/generated/package/multiversion/1.1.0/index.json

```diff
@@ -31,6 +31,9 @@
       "versions": "\u003e6.7.0"
     }
   },
+  "conditions": {
+    "kibana.version": "\u003e6.7.0"
```

Comment:
I wonder why some fields are in such encoded format and some are not: `kibana.version: ">6.7.0  <7.6.0"`

Reply:
The "problem" is the `>` char which gets converted. But I tested this quite some time ago and it works as expected even though it is converted.